clients/ts/signalr/src/XhrHttpClient.ts

```diff
@@ -31,7 +34,7 @@ export class XhrHttpClient extends HttpClient {
             const xhr = new XMLHttpRequest();
 
             xhr.open(request.method!, request.url!, true);
-            xhr.withCredentials = true;
+            xhr.withCredentials = !this.options.disableXhrWithCredentials;
```

Comment:
This would be `typeof this.options.withCredentials === "undefined" || this.options.withCredentials` given my comment above.